EnglishFilter/sections/specific.txt

```diff
@@ -302,6 +302,11 @@ javstream.co,files.im,readsnk.com,tokyoghoulre.com,demonslayermanga.com,thelittl
 uploadboy.com##.text-center > a[target="_blank"] > img
 101soundboards.com##.adsense_matched_content
 101soundboards.com##.ads_top_container_adsense
+||ts.icrypto.media/_offer_promt/
+icrypto.media#?#.head_2:has(> a[href^="https://cryptofans.news"])
+icrypto.media##.aside_land_offer
+icrypto.media##div[data-rtadcontent]
+cryptofans.news##ins[style^="display"][style*="width:300px;"]
```

Comment:
`cryptofans.news##ins[style^="display"][style*="width:300px;"]`
better use this selector `ins[class^="bmadblock-"]`